build/charts/antrea/crds/packetcapture.yaml

```diff
@@ -0,0 +1,196 @@
+        - jsonPath: .spec.destination.ip
+          description: The IP address of the destination.
+          name: Destination-IP
+          type: string
+          priority: 10
+        - jsonPath: .spec.timeout
+          description: Timeout in seconds.
+          name: Timeout
+          type: integer
+          priority: 10
+        - jsonPath: .metadata.creationTimestamp
+          name: Age
+          type: date
+      schema:
+        openAPIV3Schema:
+          type: object
+          required:
+            - spec
+          properties:
+            spec:
+              type: object
+              required:
+                - fileServer
+                - source
+                - captureConfig
+                - destination
+              anyOf:
+                - properties:
+                    source:
+                      required: [pod]
+                - properties:
+                    destination:
+                      required: [pod]
+              properties:
+                source:
+                  type: object
+                  nullable: true
+                  oneOf:
+                    - required:
+                      - pod
+                    - required:
+                      - ip
+                  properties:
+                    pod:
+                      type: string
+                    namespace:
+                      type: string
+                    ip:
+                      type: string
+                      oneOf:
+                        - format: ipv4
+                        - format: ipv6
+                destination:
+                  type: object
+                  nullable: true
+                  oneOf:
+                    - required:
+                      - pod
+                    - required:
+                      - ip
+                    - required:
+                      - service
+                  properties:
+                    pod:
+                      type: string
+                    service:
+                      type: string
+                    namespace:
+                      type: string
+                    ip:
+                      type: string
+                      oneOf:
+                        - format: ipv4
+                        - format: ipv6
+                packet:
+                  type: object
+                  x-kubernetes-validations:
+                    - rule: "(self.ipFamily == 'IPv4' && self.protocol != 'IPv6-ICMP' && self.protocol != 58) || (self.ipFamily == 'IPv6' && self.protocol != 'ICMP' && self.protocol != 1) "
+                      message: "packet.ipFamily is incompatiable with packet.protocol"
+                  properties:
+                    ipFamily:
+                      type: string
+                      enum: [IPv4, IPv6]
+                      default: IPv4
+                    protocol:
+                      x-kubernetes-int-or-string: true
+                      enum: [ICMP, TCP, UDP, IPv6-ICMP, 1, 6, 17, 58]
```

Comment:
it feels like for packet capture, we don't need to have a fixed list of protocols? we could filter on an arbitrary protocol value
also SCTP is missing

Reply:
i didn't add support for SCTP yet. will try to work on that. or maybe this is not a must-have feature in this time.

Reply:
My main point was not about SCTP specifically, but about whether we really need to support a _fixed_ list of protocols, as opposed to supporting a common set of names, but also letting users provide an arbitrary protocol number.

Reply:
sure. remove this for now and i will double check the impl code and add more tests.

Reply:
https://github.com/antrea-io/libOpenflow/blob/f8e71bfb83160a95d2ec6ca53bf55e6f821593e9/protocol/ip.go#L17

Not sure if i missed anything here, but the type code defined in this file doesn't have SCTP.